tests/http/test_17_ssl_use.py

```diff
@@ -77,9 +77,8 @@ def test_17_02_sslinfo_reconnect(self, env: Env, httpd, nghttpx, tls_max, repeat
             exp_resumed = 'Initial'  # rustls does not support sessions, TODO
         if env.curl_uses_lib('bearssl') and tls_max == '1.3':
             pytest.skip('BearSSL does not support TLSv1.3')
-        if env.curl_uses_lib('mbedtls') and tls_max == '1.3' and \
-                not env.curl_lib_version_at_least('mbedtls', '3.6.0'):
-            pytest.skip('mbedtls TLSv1.3 support requires at least 3.6.0')
+        if env.curl_uses_lib('mbedtls') and tls_max == '1.3':
+            pytest.skip('mbedtls TLSv1.3 session resume not working in 3.6.0')
```

Comment:
I think this description needs clarification/alteration... The version check is removed, but the description still mentions version 3.6.0. And more iterestingly, is tls1.3 session resumption not working in mbedtls? Or does libcurl not yet support mbedtls's tls1.3 session resumption?

Reply:
curl supports session setting in mbedtls, but our test server reports that resumption has not happened. That is why I excluded this test here.

If that is a shortcoming in mbedtls or if we miss something special for TLSv1.3 and mbedtls here remains to be investigated.